labels/lib/model/hex_color.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:equatable/equatable.dart';
+
+class HexColor with EquatableMixin {
+  final String value;
+
+  HexColor(this.value);
```

Comment:
should we precheck the color must be start with `#`?

Reply:
Added